test/unit/index.test.ts

```diff
@@ -23,10 +23,21 @@
  --------------
  ******/
 
-import index from '../../src/index'
-import Config from '../../src/shared/config'
+import index from '~/index'
+import Config from '~/shared/config'
 import { Server } from '@hapi/hapi'
 
+import { Transactions } from '~/domain/thirdpartyRequests'
+import Logger from '@mojaloop/central-services-logger'
+import MockData from '../unit/data/mockData.json'
+
+const mock_forwardTransactionRequest = jest.spyOn(Transactions, 'forwardTransactionRequest')
+const mock_loggerPush = jest.spyOn(Logger, 'push')
+const mock_loggerError = jest.spyOn(Logger, 'error')
+const mock_data = JSON.parse(JSON.stringify(MockData))
```

Comment:
Could we use an expand operator instead? 

```ts
const mock_data = { ... MockData }
```

Reply:
That may not work though - since nested objects won't be cloned.

Reply:
Ok